x-pack/plugins/ml/server/routes/job_service.ts

```diff
@@ -272,6 +272,41 @@ export function jobServiceRoutes({ router, routeGuard }: RouteInitialization) {
     })
   );
 
+  /**
+   * @apiGroup JobService
+   *
+   * @api {post} /api/ml/jobs/jobs_for_export Get job
```

Comment:
Why did you go with `jobs_for_export` here, rather than say `jobs_for_cloning`? Is the thinking that we'd use this functionality when we add the ability to export / import job configs? Just wondering as the comment below also just refers to `cloning`.

Reply:
Updated here https://github.com/elastic/kibana/pull/88898/commits/37e73ae597cc4acd9db6db59a17ec893799d240d